OptimizelySDK/OptlyConfig/OptimizelyConfigService.cs

```diff
@@ -0,0 +1,238 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using OptimizelySDK.Entity;
+using OptimizelySDK.Utils;
+
+
+namespace OptimizelySDK.OptlyConfig
+{
+    public class OptimizelyConfigService
+    {
+        private static OptimizelyConfigService Instance;
+
+        private OptimizelyConfigService() { }
+
+        public static OptimizelyConfigService getInstance()
```

Comment:
Add `static` constructor and do all instance intiailization there. 
add `instance` as static property.

Reply:
@jaeopt Please suggest alternate solution if singleton doesn't make sense here. I really liked Mike's approach in java-sdk. 
`OptimizelySerivce(ProjectConfig).getOptimizelyConfig()` looks pretty cool. What do you suggest?

Reply:
Yes, that looks good to me too!